ground/gcs/src/plugins/coreplugin/generalsettings.cpp

```diff
@@ -189,10 +189,14 @@ void GeneralSettings::readSettings(QSettings* qs)
     m_proxyHostname = qs->value(QLatin1String("proxyhostname"),m_proxyHostname).toString();
     m_proxyUser = qs->value(QLatin1String("proxyuser"),m_proxyUser).toString();
     m_proxyPassword = qs->value(QLatin1String("proxypassword"),m_proxyPassword).toString();
-    m_forumUser = qs->value(QLatin1String("forumuser"),m_forumUser).toString();
-    m_forumPassword = qs->value(QLatin1String("forumpassword"),m_forumPassword).toString();
-    m_aircraft = qs->value(QLatin1String("aircraft"),m_aircraft).toString();
-    m_observations = qs->value(QLatin1String("observations"),m_observations).toString();
+    m_observations = qs->value(QLatin1String("observations"), "").toString();
+    m_vehicle = qs->value(QLatin1String("vehicle"), "").toString();
+    m_board = qs->value(QLatin1String("board"), "").toString();
+    m_weight = qs->value(QLatin1String("weight"), 0).toInt();
```

Comment:
Yea.. if you enter something like `225mtm` here, it gets turned into `0`, just confirmed.
![1449130090](https://cloud.githubusercontent.com/assets/6873/11555026/f9463c2e-9a01-11e5-9e6e-e52bae5b5899.png)


```
Flight controller:	Brain
Firmware tag:		next_fujin
Firmware commit:	dc4d0a8
Firmware date:		20151203 07:23 UTC

Aircraft description:
Type:			QuadX
Weight (AUW):	0 g
Size:			0
Battery:		1S
Motors:			
ESCs:			

Observations:


Measured properties:
		Gain		Bias		Noise
Roll:	10.5324		-0.00423568	24.5134
Pitch:	10.2604		0.00246422	20.4178
Tau:	14.7 ms

Tuning aggressiveness:
Damping:			1.1
Noise sensitivity:	1 %
Natural frequency:	13.5

Computed values:
			RateKp		RateKi		RateKd
Roll:		0.00197253	0.0135056	2.13666e-05
Pitch:		0.00258912	0.0177273	2.80456e-05
Outer Kp:	12.592		-			-
Derivative cutoff:	46
```